zebra-instructions.md

```diff
@@ -101,7 +101,7 @@ Install dependencies:
     sudo apt-get install redis
     redis-server
     ```
-2. Install and activate [`nodenv`](https://github.com/nodenv/nodenv#installation) or [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating)
+2. Install and activate a node version manager (e.g. [`nodenv`](https://github.com/nodenv/nodenv#installation) or [`nvm`](https://github.com/nvm-sh/nvm#installing-and-updating))
```

Comment:
Can you please add a `.nvmrc` to the project root directory, so that users don't have to manually set the node version?
https://github.com/nvm-sh/nvm#nvmrc

This can happen in another PR.

Reply:
Good idea, added in [`8b0cf93`](https://github.com/ZcashFoundation/s-nomp/pull/3/commits/8b0cf93ed3408880502728bbd1fe191e65e630e9).